UI kit

denisov-ie opened the pull request:
На проверку
Посмотреть: https://denisov-ie.github.io/ui-kit/

denisov-ie committed:
UI Kit first commit

denisov-ie committed:
поправил ридми

denisov-ie committed:
1. Добавил курсор пойнтер везде, где ожидается
2. Замочек теперь не кнопка
3. Лейблы теперь фокусят на инпуты
4. Серая обводка у черных кнопок
5. Все картинки в одном месте, вызываются по айди
6. Везде убраны лишние дивы
7. Ширина инпута и серчбара теперь подстраивается по блок, в котором они находятся
8. У кнопок теперь ширина по содержимому
9. Правка именований согласно БЭМ
10. Стили кнопок связаны для уменьшения кол-ва модификаторов
11. Ссылка на просмотр - https://denisov-ie.github.io/ui-kit/
12. Теперь можно переключать темы

denisov-ie committed:
Ремонт слетевшего шрифта

denisov-ie committed:
Ссылка на шрифт должна быть https

Muslik commented on index.html:
```diff
@@ -0,0 +1,582 @@
+            <!--Wrapper for separating button blocks for a better view, it is not part of the button-->
+            <div class="button__wrapper">
+                <!--Button with text and icon, blue colored, medium size-->
+                <div class="button button_icon_text button_blue button_medium">
```
Зачем эта обертка в виде div? Уберите div, перенесите классы в сам button

Muslik replied:
button_icon_text ненужный класс

denisov-ie replied:
fixed

Muslik commented on index.html:
```diff
@@ -0,0 +1,582 @@
+                    </label>
+                    <label>
+                        <li class="dropdown__item">
+                            <div class="checkbox checkbox_with_text">
```
checkbox_with_text - класс не по бэм
checkbox_with-text - по бэм

denisov-ie replied:
fixed

Muslik commented on index.html:
```diff
@@ -0,0 +1,582 @@
+                    </label>
+                    <label>
+                        <li class="dropdown__item">
+                            <div class="radio radio_with_text_only">
```
radio_with_text_only - класс не по БЭМ.
Пробел между словами через "-" -> radio_with-text-only

denisov-ie replied:
fixed

Muslik reviewed (approved):
Ставлю авансом в надежде что исправите ошибки.

denisov-ie committed:
1. Убрал лишние div у кнопок
2. Убрал ненужные классы, пришлось оставить модификатор "только с иконкой" для кнопки
3. Переименования классов согласно БЭМ
4. Еще раз проверил цвет крестиков на макете - цвет тот же, что указан в css - #BAD8F5 (--color-day-light-blue)
